sdk/iot/core/inc/az_iot_provisioning_client.h

```diff
@@ -0,0 +1,310 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+/**
+ * @file az_iot_provisioning_client.h
+ *
+ * @brief definition for Azure Device Provisioning Service device-client SDK.
+ */
+
+#ifndef _az_IOT_PROVISIONING_CLIENT_H
+#define _az_IOT_PROVISIONING_CLIENT_H
+
+#include <az_result.h>
+#include <az_span.h>
+#include <stdbool.h>

```

Comment:
use a single empty line between az_ includes and the ones from stdlib, then auto-format (Ctrl+K+D in VS), and it is going to place az_span before the az_result. without an empty line, it would sort both our headers and stdlib headers alphabetically, placing <assert.h> before <az_span.h>.